src/tests/test_piranha_java.rs

```diff
@@ -59,13 +59,55 @@ create_rewrite_tests! {
   test_new_line_character_used_in_string_literal:  "new_line_character_used_in_string_literal",   1;
   test_insert_field_and_initializer:  "insert_field_and_initializer", 1;
   test_consecutive_scope_level_rules: "consecutive_scope_level_rules",1;
+  test_user_option_delete_if_empty: "user_option_delete_if_empty", 1;
+  test_user_option_do_not_delete_if_empty : "user_option_do_not_delete_if_empty", 1, delete_file_if_empty=false;
 }
 
 create_match_tests! {
   JAVA,
   test_java_match_only: "structural_find", 20;
 }
 
+#[test]
+
+fn test_user_option_delete_consecutive_lines() {
+  initialize();
+  let _path = PathBuf::from("test-resources")
+    .join(JAVA)
+    .join("user_option_delete_consecutive_lines");
+  let temp_dir = copy_folder_to_temp_dir(&_path.join("input"));
+
+  let piranha_arguments = piranha_arguments! {
+    path_to_codebase = temp_dir.path().to_str().unwrap().to_string(),
+    path_to_configurations = _path.join("configurations").to_str().unwrap().to_string(),
+    language= JAVA.to_string(),
+    delete_consecutive_new_lines= true,
```

Comment:
```suggestion
    delete_consecutive_new_lines = true,
```

Just for consistency between all four options being passed. Unless there is any significance of `a = b` vs `a= b` somehow that I am missing.